### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by StevenYCChou**
Question: should we also backtrace to when the repo was created?

I don't know the answer of the question, so I'll just update the license for users start to use this repositories after this commit, as I think it's reasonable.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@ Apache License
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2020- Google, LLC.
```
Comment (igorpeshansky) [inline LICENSE]: The repo was [created in 2014](https://github.com/GoogleCloudPlatform/fluentd-catch-all-config/commits/master/LICENSE), so this should be `Copyright 2014 Google LLC` (or, if you prefer, `Copyright 2014-2020 Google LLC`).

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
The repo was first created in 2014: https://github.com/GoogleCloudPlatform/fluentd-catch-all-config/commits/master/LICENSE. Since that fact is easily traceable through the commit history, we should use the earliest year.

File: LICENSE (hunk 1 above)
Comment (a-robinson) [reply LICENSE:189]: Yup, 2014 would be best. I was working for Google when I created it, so no worries there.

**Review by a-robinson — CHANGES_REQUESTED** [review]
Yup, 2014 would be best. I was working for Google when I created it, so no worries there.

**Comment by StevenYCChou** [comment]
Updated, please take another look. Thanks!

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**